### PR TITLE
Finished writing the Food Entity class

### PR DESCRIPTION
**Pull request by Enidx**
and its attribute and constructors. Please review.

### CODE REVIEW

File: src/main/java/Food [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    public double sugar;
+
+    public String foodType;
+    public String id;
+    public boolean vegFriendly;
+    public int nutrientScore;
+
+    /**
+     *
+     * @param calories is a percentage of recommended c
```
Comment (naomianne) [inline src/main/java/Food:21]: this sentence doesn't seem to be finished?

File: src/main/java/Food [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    public double proteins;
+    public double fat;
+    public double sugar;
+
+    public String foodType;
+    public String id;
+    public boolean vegFriendly;
+    public int nutrientScore;
+
+    /**
```
Comment (naomianne) [inline src/main/java/Food:19]: maybe write something like "construct a food object" since it's a constructor

**Review by naomianne — COMMENTED** [review]
looks good overall

File: src/main/java/Food (hunk 1 above)
Comment (Enidx) [reply src/main/java/Food]: Yepyep sorry, fixed!

**Pull request merged**